crates/nu-table/src/table.rs

```diff
@@ -847,7 +847,7 @@ fn truncate_columns_by_head(
     widths
 }
 
-/// The same as [`tabled::peaker::PriorityMax`] but prioritizes left columns first in case of equal width.
```

Comment:
Same, if this type still exists in the [`tabled` crate](https://docs.rs/tabled) maybe worth still mentioning it as a literal text without link.

Reply:
I thought `tabled` was a module, not the crate, my bad, fixed